bin/eslint-formatters-calypso/README.md

```diff
@@ -0,0 +1,82 @@
+# eslint-formatters-calypso
+
+A set of formatters to be used by ESLint and utilities to build new formatters upon.
+
+Formatters available:
+
+* `lines-modified`: downgrade ESLint `errors` to `warnings`, except for lines modified in the topic branch.
+* `lines-modified-and-specific-rules`: downgrade ESLint `errors` to `warnings`, except for lines modified in the topic branch and the rules in the `.eslines.json` file.
+* `parsing-errors`: takes the ESLint original report and outputs a new one containing only `parsing errors`.
+* `eslines`: a wrapper formatter. If the branch active is `master`, would return the results of `parsing-errors` formatter. Otherwise, would return the results of `lines-modified-and-specific-rules` formatter.
+
+### How to install
+
+From npm:
+
+	npm install eslint-formatters-calypso
```

Comment:
We need to choose whether this will be a separate `npm` package and live in a separate repository or live inside Calypso. I am ok with either, but if we choose the latter, I don't think we would need `package.json`, `.gitignore`, or instructions like the above.